test/TestCommandLineInvocationSameAsConfig.py

```diff
@@ -74,3 +74,19 @@ def test_path_from_cli_depend_on_cwd(base_arguments, monkeypatch):  # Issue 572
 
     assert 'test/test' not in options1.exclude_paths[0]
     assert 'test/test' in options2.exclude_paths[0]
+
+
+def test_invalid_config_file(base_arguments):
+    """Ensures invalid config_file produces error code 2."""
+    with pytest.raises(SystemExit) as excinfo:
```

Comment:
Have you tried 
```suggestion
    with pytest.raises(SystemExit, match='^2$'):
```
(not sure if it'll work, but it'd be cleaner if yes)